### ISSUE TITLE
EPSImageFile does not handle Ghostscript 9.10 responses correctly

### ISSUE THREAD

**Issue by euangoddard** [python-pillow/Pillow#701]
When I try to open an EPS file, I get an error:
``` python
>>> from PIL import Image
>>> image = Image.open('/home/euan/Desktop/U.S.A Flags_Sample.eps')
>>> image.load()
GPL Ghostscript 9.10: Unrecoverable error, exit code 1
```

The EPS file seems fine and moreover, if I skip the check [here](https://github.com/python-pillow/Pillow/blob/master/PIL/EpsImagePlugin.py#L134) the image is correctly loaded and can be manipulated.

This suggests that exit code 1 is a success from GhostScript and PIL could proceed processing.

I am using GhostScript 9.10 on 64-bit Ubuntu 14.04, running Pillow 2.4.0

**Comment by radarhere** [comment]
Just fyi, I think I found the file that was used here - http://soccerwow.com/art/holder/wavyUSFlag/084_objects/U.S.A%20Flags_Sample/U.S.A%20Flags_Sample.eps

I can replicate the problem with it at least.

**Comment by radarhere** [comment]
From my testing, this has been fixed by #1478
